import/neo4j

```diff
@@ -27,6 +27,7 @@ import os
 import warnings
 
 DEFAULT_ENDPOINT =  'http://neo4j:insecure@localhost:7474/db/data'
+wipeDatabase = True
```

Comment:
It was, because that's the way I've been importing data... but it makes more sense to make it a non-default command line parameter.